packages/material-ui/src/Paper/Paper.js

```diff
@@ -91,7 +86,17 @@ Paper.propTypes = {
    * Shadow depth, corresponds to `dp` in the spec.
    * It accepts values between 0 and 24 inclusive.
    */
-  elevation: PropTypes.number,
+  elevation: chainPropTypes(PropTypes.number, (props) => {
+    const { classes, elevation } = props;
+    // in case `withStyles` fails to inject we don't need this warning
+    if (classes === undefined) {
```

Comment:
I'm surprised this case can be reached.

Reply:
Most likely never. Was simple to guard against this while writing though.